src/main/java/net/neoforged/neoforge/client/model/generators/ModelBuilder.java

```diff
@@ -480,6 +480,24 @@ public ElementBuilder allFaces(BiConsumer<Direction, FaceBuilder> action) {
             return this;
         }
 
+
+         /**
+         * Modify all <em>possible</em> faces except provided in second argument dynamically using a function, creating new
+         * faces as necessary.
```

Comment:
"Modify all faces, except those provided in the second argument, dynamically using a function, creating new faces as necessary."

The description of allFaces() is already a bit wonky; your additions didn't make it better. That "all <em>possible</em>" should just be "<em>all</em>".  There are no "impossible faces"---the Direction enum is finite, all is all.

Reply:
You are right, but sometimes may be situations, when some faces is not needed for model part. Like here:

```java
element().
     from(7, 6, 7).
     to(9, 7, 9).
     face(Direction.NORTH).end().
     face(Direction.SOUTH).end().
     face(Direction.WEST).end().
     face(Direction.EAST).end().
     face(Direction.UP).end().
     faces((face, builder) ->
     {
           builder.texture("#button");
           if (face != UP)
                builder.uvs(0, 8, 8, 12);
           else
                builder.uvs(0, 0, 8, 8);
      }).
      end()
```

In this situation, `Direction.DOWN` is useless for me, so to make the model a bit lighter and write less code with the new method, it will be easier to write:

```java
element().
     from(7, 6, 7).
     to(9, 7, 9).
     allFacesExcept((face, builder) ->
     {
           builder.texture("#button");
           if (face != UP)
                builder.uvs(0, 8, 8, 12);
           else
                builder.uvs(0, 0, 8, 8);
      }, EnumSet.of(Direction.DOWN)).
      end()
```


It might make sense to swap the list of exceptions and actions, but otherwise, everything is quite clear.

Reply:
I was only suggesting a better javadoc text and ranting about the text of the existing allFaces. The concept was fine.